Lib/Inc/CJsonParser.h

```diff
@@ -26,6 +26,7 @@ class CJsonParser : public ICommand
     bool parse(const etl::string<MAX_STRING_SIZE> &string);
     const etl::string<MAX_STRING_SIZE> *getName() const;
     unsigned int getArgumentCount() const;
+    const etl::string<MAX_STRING_SIZE> *getStringArgument() const;
```

Comment:
Why are you returning pointer? I thought returning by reference was the preferred way for etl::string. Not criticims, just curious.

Reply:
Just for consistency. There's a getName() method that does almost the same in IComChannel and it returns a pointer. 
I created a ticket the other day to change pointers for references when possible.